Cubical/Categories/Colimits/Coproduct.agda

```diff
@@ -0,0 +1,43 @@
+-- Binary coproducts
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Colimits.Coproduct where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Data.Sigma.Base
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Prelude
+open import Cubical.HITs.PropositionalTruncation.Base
+
+private
+  variable
+    ℓ ℓ' : Level
+
+module _ (C : Category ℓ ℓ') where
+  open Category C
+
+  module _ {x y x+y : ob}
+           (i₁ : Hom[ x , x+y ])
+           (i₂ : Hom[ y , x+y ]) where
+
+    isCoproduct : Type (ℓ-max ℓ ℓ')
+    isCoproduct = ∀ (z : ob) (f₁ : Hom[ x , z ]) (f₂ : Hom[ y , z ]) →
+        ∃![ f ∈ Hom[ x+y , z ] ] (i₁ ⋆ f ≡ f₁) × (i₂ ⋆ f ≡ f₂)
+
+    isPropIsCoproduct : isProp (isCoproduct)
+    isPropIsCoproduct = isPropΠ3 λ z f₁ f₂ → isPropIsContr
+
+
+  record Coproduct (x y : ob) : Type (ℓ-max ℓ ℓ') where
```

Comment:
I would call this as well as the file `BinCoproduct` as it's just the binary version. I would prefer to reserve `Coproduct` for  general indexed coproducts